### ISSUE TITLE
button does not infer aria-label properly 

### ISSUE THREAD

**Issue by ThomasBurleson** [angular/material#6731]
mbButton does not use $mdAria correct to set the aria-label based on the label/content text.

**Comment by ThomasBurleson** [comment]
Should use:

```js
    var elementHasText = node.textContent.trim();
    $mdAria.expect(element, 'aria-label', elementHasText);
```

**Comment by ThomasBurleson** [comment]
Fixed with SHA 571634065a6a8af570ce308ceb6ba829553eeaa9
